### ISSUE TITLE
Create Footer for all Pages

### ISSUE THREAD

**Issue by devppratik** [Developer-Student-Clubs-VSSUT-Burla/CEM-frontend#11]
A footer component needs to be added that will be used across all pages. 
Few requirements are 

- 3 or 4 column footer
-  Fully Responsive
-  Must include logo Put VSSUT-Events for now )

Sample is attached for reference
![Screenshot_20211013_105221](https://user-images.githubusercontent.com/56730201/137072287-d9729faf-4a70-4e95-ab4d-c7043fa25347.png)

**Comment by MujahidAli954** [comment]
I want  to work on this task if you assign me

**Comment by chandrasekhar2039** [comment]
> I want to work on this task if you assign me

Yeah go for it. After you done working on it attach screenshot of your work while opening a pr.
